auto-generated/api-testing/vfadd.c

```diff
@@ -244,3 +244,243 @@ vfloat64m8_t test_vfadd_vf_f64m8_m(vbool8_t mask, vfloat64m8_t op1, float64_t op
   return __riscv_vfadd_vf_f64m8_m(mask, op1, op2, vl);
 }
 
+vfloat16mf4_t test_vfadd_vv_f16mf4_rm(vfloat16mf4_t op1, vfloat16mf4_t op2, size_t vl) {
+  return __riscv_vfadd_vv_f16mf4_rm(op1, op2, FRM_RNE, vl);
```

Comment:
We can't use `FRM_RNE`. It needs to be prefixed with underscores. Probably needs to be `__FRM_RNE` or `__RISCV_FRM_RNE`

Reply:
Updated the enum to 

```c
enum __RISCV_FRM {
  __RISCV_FRM_RNE = 0,
  __RISCV_FRM_RTZ = 1,
  __RISCV_FRM_RDN = 2,
  __RISCV_FRM_RUP = 3,
  __RISCV_FRM_RMM = 4,
};

```